tests/phpunit/CRM/Utils/TokenConsistencyTest.php

```diff
@@ -470,8 +470,8 @@ protected function getExpectedEventTokenOutput(): string {
 Emerald City, Maine 90210
 
 $ 50.00
-' . CRM_Utils_System::url('civicrm/event/info', NULL, TRUE) . '&amp;reset=1&amp;id=1
-' . CRM_Utils_System::url('civicrm/event/register', NULL, TRUE) . '&amp;reset=1&amp;id=1
+' . CRM_Utils_System::url('civicrm/event/info', NULL, TRUE) . '&reset=1&id=1
+' . CRM_Utils_System::url('civicrm/event/register', NULL, TRUE) . '&reset=1&id=1
```

Comment:
I'm confused by how changing the pdf for contribution thank you letters affects the event urls for scheduled event reminders? I admit I haven't followed it thru yet but maybe you already know the answer.

Reply:
ug I've forgotten - I think it must be this line

https://github.com/civicrm/civicrm-core/pull/21524/files#diff-1321b0fc6aa7ca714e847f911931f138fb0f290877a5557bab65725e77cc3157R189

Reply:
oh yeah it is - that line is saying 'this is the html version I'm giving you' - which results in less change to the pdf output

Reply:
&amp;Thanks.